### ISSUE TITLE
Unable to publish resume in my name:  throw new Error('Cannot switch to old mode now.');

### ISSUE THREAD

**Issue by kaldrenon** [jsonresume/resume-cli#138]
Full error output is below.

I am trying to publish to the account for my username, and it fails. I have confirmed that there are no errors when I export to html or when I publish anonymously, but when I attempt to publish to my account, I get the output below.

```
→ resume publish
  running validation tests on resume.json ...

  ✓ Passed all validation tests.

To publish your resume at http://jsonresume.org type the command resume publish


             PUBLISH MENU
     ------------------------------
     Publish to your account. <--
     Publish as guest.
     Publish to your account (password access).
     Publish as guest (password access).
     ------------------------------
     EXIT

     To acquire account details, press ^C and type: `resume register`
     Or just publish as guest.

SELECTED: Publish to your account.

To acquire login details use the command: `resume register`
If you already have an account, enter your login details below:

_stream_readable.js:748
    throw new Error('Cannot switch to old mode now.');
          ^
Error: Cannot switch to old mode now.
    at emitDataEvents (_stream_readable.js:748:11)
    at ReadStream.Readable.resume (_stream_readable.js:733:3)
    at new Interface (readline.js:142:9)
    at Object.exports.createInterface (readline.js:39:10)
    at read (/usr/local/lib/node_modules/resume-cli/node_modules/read/lib/read.js:45:23)
    at /usr/local/lib/node_modules/resume-cli/lib/publish/index.js:59:21
    at Menu.<anonymous> (/usr/local/lib/node_modules/resume-cli/lib/publish/menu.js:33:7)
    at Menu.emit (events.js:98:17)
    at Menu.ondata [as _ondata] (/usr/local/lib/node_modules/resume-cli/node_modules/terminal-menu/index.js:215:18)
    at DestroyableTransform._transform (/usr/local/lib/node_modules/resume-cli/node_modules/terminal-menu/index.js:50:18)
    at DestroyableTransform.Transform._read (/usr/local/lib/node_modules/resume-cli/node_modules/terminal-menu/node_modules/through2/node_modules/reada
ble-stream/lib/_stream_transform.js:184:10)

**Comment by PeterDaveHello** [comment]
What's your account ID @kaldrenon ?

**Comment by kaldrenon** [comment]
Username is `andrewfallows`

**Comment by yvettecook** [comment]
I'm getting this issue as well - is there a fix I can do myself? If not, my username is yvettecook

**Comment by PeterDaveHello** [comment]
ping @rolandnsharp @thomasdavis for help.

**Comment by santiph** [comment]
I've solved it by updating Node to the latest downloaded version on the official page.

**Comment by rolandnsharp** [comment]
Hmmm, This is my mistake, I should try to support old versions of node. I had to update to get those menus working with that latest version of Windows.

**Comment by santiph** [comment]
I am using the latest available version of Node downloaded from its website
two days ago...
Is there a newer version around I should be using?

**Comment by kdmcshane** [comment]
Having similar issues myself. I know I have the most updated Node.js installed because I just installed it today on my Windows 7 OS. I had no problem registering a page on JsonResume but when I go to publish it to my account in "Node.js command prompt" using "publish resume" after selecting "Publish to your account" I get a message saying:

SELECTED: Publish to your account.

To acquire login details use the command: 'resume register'
If you already have an account, enter your login details below:
email: <(shows my email I used)>

Then basically no matter what button I push after that, it goes back to the default command prompt where I started before I even entered any command. For some reason I have absolutely no problems publishing it "as guest" though.
